### ISSUE TITLE
Why items is Ref[]

### ISSUE THREAD

**Issue by Sozialarchiv** [archival-IIIF/presentation-builder#1]
@kerim1 

https://github.com/archival-IIIF/presentation-builder/blob/master/v3/Base.ts#L39

How can I handle manifest files with an items attribute, that contains images for example?

**Comment by kerim1** [comment]
I am not sure I understand the question? Can you rephrase it? Also see https://github.com/archival-IIIF/server/blob/e1e225c9525d82d35cf933af7942d2a35f9b3782/src/presentation/builder/DigitalBorn.ts

**Comment by Sozialarchiv** [comment]
For the test-server I wrote a function that transform V3 to V2. The function is not very perfect, but it saves a lot of code.

The problem is here:
https://github.com/archival-IIIF/test-server/blob/master/src/lib/Transform.ts#L77

On iterating the items of an v3/Manifest (e.g. a Manifest with an image), typescript wont let you get manifest.items[0].items[0], because Ref has no attribute items.

So I guess items should just in case of the Collection class Ref[] and otherwise Base[].
